fury/tests/test_primitive.py

```diff
@@ -1,19 +1,29 @@
 import numpy as np
 import numpy.testing as npt
-import fury.primitive as fp
+import fury.primitive as fpz
 
 
 def test_vertices_primitives():
+    # Tests the default vertices of all the built in primitive shapes.
     l_primitives = [(fp.prim_square, (4, 3)),
-                    (fp.prim_box, (8, 3))]
+                    (fp.prim_box, (8, 3)),
+                    (fp.prim_star, np.shape(fp.prim_star()),
+                     -3, 3, -0.666666666667),
+                    (fp.prim_rhombicuboctahedron,
+                     np.shape(fp.prim_rhombicuboctahedron()), -4, 4, 0)]
 
-    for func, shape in l_primitives:
+    for func, shape, min, max, mean in l_primitives:
```

Comment:
`min, max, mean` are predefined by python so it is better to avoid this variable name. Can you change them? Just put `e_min, e_max, e_mean` the e is for expected